resources/cmake/ConnextDdsAddExamplesSubdirectories.cmake

```diff
@@ -0,0 +1,84 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
```

Comment:
I think this license is not correct, see for example: https://github.com/rticommunity/rticonnextdds-cmake-utils/blob/main/cmake/Modules/ConnextDdsCodegen.cmake

Reply:
I'm creating a task to review the license